Gemfile

```diff
@@ -31,10 +30,6 @@ group :development do
   gem 'avro_turf', '>= 0.8.0', require: false
   gem 'heroku_rails_deploy', '>= 0.4.1', require: false
   gem 'overcommit'
-  gem 'spring'
```

Comment:
I removed Spring as it is no longer recommended for Rails: https://github.com/rails/rails/pull/42997